### PR TITLE
fix(sdk-lib-mpc): fix 0 values attack on range proof

### PR DESCRIPTION
**Pull request by zhongxishen**
TICKET: HSM-159

### CODE REVIEW

**Review by aarvay — COMMENTED** [review]
flushing due to lack of context. also unsure why custody-exp is part of the codeowners

File: modules/sdk-lib-mpc/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "types": "./dist/src/index.d.ts",
   "scripts": {
     "test": "yarn unit-test",
-    "unit-test": "nyc -- mocha",
+    "unit-test": "nyc -- mocha --recursive test",
```
Comment (zahin-mohammad) [inline modules/sdk-lib-mpc/package.json:9]: Could add context on to why we need these flags + the etension change in the .mocharc?

**Review by zahin-mohammad — COMMENTED** [review]
typescript changes lgtm, just had a comment on the config changes

File: modules/sdk-lib-mpc/package.json (hunk 1 above)
Comment (zhongxishen) [reply modules/sdk-lib-mpc/package.json:9]: `spec: ['test/unit/*.ts']` didn't run tests under sub-directories of `unit/`.

it's after the same change from #3807

Comment (zahin-mohammad) [reply modules/sdk-lib-mpc/package.json:9]: I think you can just change the spec to be. Is there a reason to have `.js` files here?
```
['test/unit/**/*.ts']
```

Comment (zahin-mohammad) [reply modules/sdk-lib-mpc/package.json:9]: WIth this the `--recursive` flag might be redundant?

Comment (zhongxishen) [reply modules/sdk-lib-mpc/package.json:9]: I don't disagree with `['test/unit/**/*.ts']`, but with #3807 already in `master`, it's better to be consistent IMO.

**Review by johnoliverdriscoll — APPROVED** [review]

**Pull request merged**